go.mod

```diff
@@ -1,24 +1,77 @@
 module github.com/openshift/elasticsearch-operator
 
-go 1.16
+go 1.17
 
-// Pinned to kubernetes-1.21.3 and openshift release-4.8
 require (
-	github.com/ViaQ/logerr v1.0.10
+	github.com/ViaQ/logerr v1.1.0
 	github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.2.3
 	github.com/google/go-cmp v0.5.5
 	github.com/inhies/go-bytesize v0.0.0-20151001220322-5990f52c6ad6
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.13.0
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.17.0
 	github.com/openshift/api v0.0.0-20210713130143-be21c6cb1bea
```

Comment:
Bumping to k8s 0.23.0, requires to bump this also to 4.10 via: `go get github.com/openshift/api@release-4.10`